src/validate.ts

```diff
@@ -0,0 +1,91 @@
+
+'use-strict';
+
+ /**
+ * Generates random request ID
+ */
+
+function generateRequestID(min, max){
+    
+        return Math.floor(Math.random() * (max-min) + min).toString()
+}
+    
+ /**
+ * Generating JSON request payloads for SYNC, QUERY, EXECUTE, and DISCONNECT intents
+ */
+export function generateSyncRequest(){
+    const requestId = generateRequestID(100,999);
+    return {
+        requestId,
+        inputs: [{
+          intent: "action.devices.SYNC"
+        }]
+    }
+}
+
+type CustomData = {[key: string]: any}
+
+interface QueryRequest{
+    requestId: string;
+    inputs: {
+     intent: string; payload: { devices: { id: string; customData: CustomData; }[]; };
+    }[];
+}
+
+export function generateQueryRequest(deviceIds: string[],customData: CustomData[]) : QueryRequest{
+    const requestId = generateRequestID(100,999);
+    return {
+        requestId,
+        inputs: [{
+          intent: "action.devices.QUERY",
+          payload: {
+            devices: deviceIds.map((deviceId, index) =>
+                ({id: deviceId, customData: customData[index]}))
+            }
+        }]
+    }
+}
+
+
+type params = {[key: string]: any}
```

Comment:
Can we apply the same TitleCase effect for the `params` type as well?

Reply:
Yep - changed